// try {
//   throw new Error("Custom Error")
// } catch(error) {
//   console.log(error)
// }

// console.log(new Error("Custom Error").stack)

console.trace(new Error("My Error"))